SUMMARY.md

```diff
@@ -2,13 +2,18 @@
 
 * [Get Started](README.md)
 * [Wallet Core](wallet-core/wallet-core.md)
-  * [Building](wallet-core/building.md)
-  * [Contributing](wallet-core/contributing.md)
-  * [Adding Support for a New Blockchain](wallet-core/newblockchain.md)
-  * [RPC / API Requirements](wallet-core/rpc-requirements.md)
-  * [Releasing](wallet-core/releasing.md)
-  * [WalletConsole utility](wallet-core/walletconsole.md)
-  * [Server-side](wallet-core/server-side.md)
+  * Using
+    * [Building](wallet-core/building.md)
+    * [WalletConsole utility](wallet-core/walletconsole.md)
+    * [Releasing](wallet-core/releasing.md)
+  * [Integration Guide](wallet-core/integration-guide.md)
+    * [Wallet Operations](wallet-core/wallet-operations.md)
+    * [iOS Wallet](wallet-code/ios-guide.md)
+    * [Server-Side](wallet-core/server-side.md)
+  * Extending
```

Comment:
what is extending?

Reply:
Extending with new functionality, e.g. with new blockchain.  Some devs are only using WC (wallet devs), some extend it (e.g. add new blockchain).